### DISCUSSION TITLE
Please install some packages for torch environment

### DISCUSSION THREAD

**Discussion by GuibinChen92057** [PaddlePaddle/PaddleSpatial#211] [BaiduKDDCup2022] [answered]
Dear Hosts, 
I need to install "logging", "os", "scipy", "math" and "random" in the pytorch environment.
Thx ^_^

**Comment by shaido987** [comment, marked as answer]
All these except scipy is part of the standard library in python and are automatically installed (see: https://docs.python.org/3/library/). Scipy on the other hand should be installed in the pytorch environment already (see installed packages [here](https://github.com/PaddlePaddle/PaddleSpatial/blob/main/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/preinstalled-pkgs/pytorch_env_installed_packages.md)). So you should be able to run your code in the online environment already.

(Note that I'm not part of the organizers of this competition.)
